messaging/methods/messages/getMessages.md

```diff
@@ -39,8 +39,9 @@ Authentication on this endpoint is <b>NOT</b> done via API token and secret. Ins
 | SENDING | Bandwidth is in the process of sending your messages to the downstream carrier. |
 | SENT | Bandwidth has sent the message. The downstream carrier has accepted the message. |
 | DELIVERED | Bandwidth has received a delivery receipt from the downstream carrier confirming successful delivery to the carrier or handset (when available). |
-| DLR_EXPIRED | Bandwidth did not receive the requested delivery receipt from the downstream carrier. The mobile operator or end-user device did not communicate back to the network indicating the message delivery. Currently, Bandwidth waits 2 hours for delivery receipts. |
 | FAILED | The message could not be sent or the delivery receipt received from the downstream carrier indicated the message was not deliverable. Review error codes for more information. |
+| ACCEPTED | Message was sent successfully, no DLR received. |
```

Comment:
The language I've seen for this is "Message was accepted by the customer". Not sure I like using the word "Sent" since it sounds like BW sent it, but this status is only populated for Inbound. And with the DLR received part, there may have been a DLR after the timeout window or something.

Reply:
Gotcha sounds good, updated